lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -73,7 +73,7 @@ - (instancetype)initWithContentsOfURL:(NSURL *)url {
         NSError *error;
         NSDictionary  *animationJSON = [NSJSONSerialization JSONObjectWithData:animationData
```

Comment:
Since you are checking the class of the deserialized object below we should change the object type to `id`

Something like

```
id jsonObject = [NSJSONSerialization JSONObjectWithData: animationData options:kNilOptions error:&error];

if (error || ! jsonObject || ![jsonObject isKindOfClass:[NSDictionary class]]) {
    return
}

NSDictionary * animationJSON = (NSDictionary *) jsonObject;
```

Reply:
Yep, you are right. I will make the change.

Reply:
Done.